### PR TITLE
Stapler: Upgrade webpack

### PR DESCRIPTION
**Pull request by lynnerang**
More homework copying

### CODE REVIEW

File: stapler/package.json [hunk 1]
```diff
@@ -37,42 +37,42 @@
...
+    "webpack": "^5.73.0",
+    "webpack-cli": "^4.10.0"
   },
   "dependencies": {
     "linkedin-design-tools-figma": "file:../plugin-tools",
-    "svelte": "^3.49.0",
-    "unique-names-generator": "^4.5.0"
+    "svelte": "^3.49.0"
   },
   "author": "Grant Blakeman <gblakeman@linkedin.com>"
```
Comment (gblakeman) [inline stapler/package.json:77]: I didn’t think of this reviewing the others, but we could probably update these `author` entries to the team: `Design Systems Productivity <designtools@linkedin.com>` – makes more sense than me. That email alias goes to a few folks in Design (including myself).

File: stapler/src/GUI.ts [hunk 2]
```diff
@@ -7,9 +7,9 @@ const { Messenger } = require('linkedin-design-tools-figma');
 const app = new App({
   target: document.body,
   props: {
-    currentView: 'general',
+    refreshIndicator: null,
```
Comment (gblakeman) [inline stapler/src/GUI.ts:10]: Just double-checking these default props changes are intended in this PR (and not copy/paste errors).

**Review by gblakeman — APPROVED** [review]
🛳💥

File: stapler/src/GUI.ts (hunk 2 above)
Comment (lynnerang) [reply stapler/src/GUI.ts:10]: yep they are, thanks. seems like the lint updates flagged an error that wasn't caught before.

**Review by joeyyang — APPROVED** [review]
🚢

**Pull request merged**